packages/frontend/src/components/composer/AppPickerWrapper.tsx

```diff
@@ -0,0 +1,49 @@
+import React, { useEffect } from 'react'
+import { AppPicker, AppInfo } from '../AppPicker'
+import { runtime } from '@deltachat-desktop/runtime-interface'
+import { getLogger } from '../../../../shared/logger'
+
+import styles from './styles.module.scss'
+import { BackendRemote } from '../../backend-com'
+import { selectedAccountId } from '../../ScreenController'
+
+const log = getLogger('renderer/components/composer/AppPickerWrapper')
+
+type Props = {
+  onAppSelected?: (app: AppInfo) => void
+  apps: AppInfo[]
+  setApps: (apps: AppInfo[]) => void
+}
+
+export const AppPickerWrapper = ({ onAppSelected, apps, setApps }: Props) => {
+
+  useEffect(() => {
+    const fetchApps = async () => {
+      try {
+        const response = await BackendRemote.rpc.getHttpResponse(
+          selectedAccountId(),
+          'https://apps.testrun.org/xdcget-lock.json'
+        )
+        const apps = await runtime.getJsonFromBase64(response.blob)
+        if (apps) {
```

Comment:
We could use the same function like in browser.runtime and thus implement that in the frontend, but I'm not really sure if that function is reliable.

Another solution would be to return a JSON string instead of a base64 encoded blob from backend, if the content-type is 'application/json'

Reply:
> since you didn't use atob/btoa

The solution uses atob but fixes the resulting "wrong" characters

I removed it now from runtime